src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar As it seem that your have a cheap i7 :wink: you should try this to get rid of the error:
ffmpeg64.exe -y -c:v hevc_cuvid -threads 1 -i jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv -bufsize 6M -maxrate 6M -b:v 3M -c:v h264_nvenc -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark